packages/db/src/definitions/networkGenealogies.ts

```diff
@@ -0,0 +1,124 @@
+import gql from "graphql-tag";
+
+import {Definition} from "./types";
+
+export const networkGenealogies: Definition<"networkGenealogies"> = {
+  createIndexes: [],
+  idFields: ["ancestor", "descendant"],
+  typeDefs: gql`
+    type CandidateSearchResult {
+      network: Network!
+      alreadyTried: [ID]! #will include all networks returned
+    }
```

Comment:
Stuff that relates to Networks more directly than NetworkGenealogies, like this, would probably fit better in `definitions/networks/`.

That way, this file is just the bare minimum about NetworkGenealogy, rather than including logic about what the existence of NetworkGenealogy does to Network